sparkle/src/components/DataTable.tsx

```diff
@@ -197,7 +202,9 @@ DataTable.Row = function Row({
     <tr
       className={classNames(
         "s-border-b s-border-structure-200 s-text-sm",
-        onClick ? "s-cursor-pointer" : "",
+        onClick
+          ? "s-cursor-pointer hover:s-border-b-neutral-500 hover:s-opacity-70"
```

Comment:
Maybe let's only do the border, opacity drop looks a little odd no?
The best would be to add a light grey background to the child <td>'s

Reply:
@albandum something like this?

Reply:
Much better - no need for the border change anymore
Remove the border change and mergeable 👍